modules.d/35network-wicked/module-setup.sh

```diff
@@ -1,76 +1,155 @@
+        "$dbussystem"/org.opensuse.Network.DHCP6.conf \
+        "$dbussystem"/org.opensuse.Network.Nanny.conf \
+        "$dbussystemservices"/org.opensuse.Network.AUTO4.service \
+        "$dbussystemservices"/org.opensuse.Network.DHCP4.service \
+        "$dbussystemservices"/org.opensuse.Network.DHCP6.service \
+        "$dbussystemservices"/org.opensuse.Network.Nanny.service \
+        "$systemdsystemunitdir"/wicked.service \
+        "$systemdsystemunitdir/wicked@.service" \
+        "$systemdsystemunitdir"/wickedd.service \
+        "$systemdsystemunitdir"/wickedd-auto4.service \
+        "$systemdsystemunitdir"/wickedd-dhcp4.service \
+        "$systemdsystemunitdir"/wickedd-dhcp6.service \
+        "$systemdsystemunitdir"/wickedd-nanny.service \
+        "$systemdsystemunitdir/wickedd-pppd@.service" \
+        ip pppd wicked wickedd wickedd-auto4 wickedd-dhcp4 wickedd-dhcp6 wickedd-nanny
+
+    # Enable systemd type units.
+    for i in \
+        wicked.service \
+        wicked@.service \
+        wickedd.service \
+        wickedd-auto4.service \
+        wickedd-dhcp4.service \
+        wickedd-dhcp6.service \
+        wickedd-nanny.service \
+        wickedd-pppd@.service; do
+        $SYSTEMCTL -q --root "$initdir" enable "$i"
     done
+
+    # Install the hosts local user configurations if enabled.
+    if [[ $hostonly ]]; then
+        inst_multiple -H -o \
+            "/etc/wicked/*.xml" \
+            "/etc/wicked/extensions/*" \
+            "$dbusconfdir"/org.opensuse.Network.conf \
+            "$dbusconfdir"/org.opensuse.Network.AUTO4.conf \
+            "$dbusconfdir"/org.opensuse.Network.DHCP4.conf \
+            "$dbusconfdir"/org.opensuse.Network.DHCP6.conf \
+            "$dbusconfdir"/org.opensuse.Network.Nanny.conf \
+            "$systemdsystemconfdir"/wicked.service \
+            "$systemdsystemconfdir/wicked.service/*.conf" \
+            "$systemdsystemconfdir/wicked@.service" \
+            "$systemdsystemconfdir/wicked@.service/*.conf" \
+            "$systemdsystemconfdir"/wickedd.service \
+            "$systemdsystemconfdir/wickedd.service/*.conf" \
+            "$systemdsystemconfdir"/wickedd-auto4.service \
+            "$systemdsystemconfdir/wickedd-auto4.service/*.conf" \
+            "$systemdsystemconfdir"/wickedd-dhcp4.service \
+            "$systemdsystemconfdir/wickedd-dhcp4.service/*.conf" \
+            "$systemdsystemconfdir"/wickedd-dhcp6.service \
+            "$systemdsystemconfdir/wickedd-dhcp6.service/*.conf" \
+            "$systemdsystemconfdir"/wickedd-nanny.service \
+            "$systemdsystemconfdir/wickedd-nanny.service/*.conf" \
+            "$systemdsystemconfdir/wickedd-pppd@.service" \
+            "$systemdsystemconfdir/wickedd-nanny.service/*.conf"
+    fi
+
+    # TODO replace this section with systemd units
+    #    for unit in "${wicked_units[@]}"; do
+    #        sed -i 's/^After=.*/After=dbus.service/g' "$initdir/$unit"
```

Comment:
AFAIK dependencies cannot be reset in drop-ins, and it seems the `After` must be set to `dbus.service` only (in every wicked service). If you want to avoid the sed, you will need to create custom services and install them, rather than installing those provided by the system.